java-lib/src/main/java/com/wavefront/api/agent/ValidationConfiguration.java

```diff
@@ -212,6 +218,14 @@ public ValidationConfiguration setLogAnnotationsCountLimit(int value) {
     return this;
   }
 
+  public boolean isEnableHyperlogsConvergedCsp() {
+    return enableHyperlogsConvergedCsp;
+  }
+
+  public void setEnableHyperlogsConvergedCsp(boolean enableHyperlogsConvergedCsp) {
```

Comment:
could we change return type to `ValidationConfiguration` for consistency?

Reply:
done